docs/src/wrapper.tsx

```diff
@@ -13,7 +13,13 @@ import { MarigoldTheme } from './components/MarigoldTheme';
 export const WrapPageElement: GatsbyBrowser['wrapPageElement'] = ({
   element,
   props,
-}) => <Layout {...props}>{element}</Layout>;
+}) => {
+  return (
```

Comment:
1. Dont need Fragment here
2. you can write the function a bit shorter without 'return':
`=> <Layout {...props}>{element}</Layout>;`

Reply:
See here 2. Arrow functions: https://www.benmvp.com/blog/10-modern-javascript-features-awesome-react/?utm_source=benmvp&utm_medium=email